test/parallel/test-stream-transform-end.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
```

Comment:
Spurious test? I think this was for https://github.com/nodejs/node/issues/35926?

Reply:
Thanks